CONVENTIONS.md

```diff
@@ -0,0 +1,10 @@
+# SDS SDK PY Aspect Model loader Code Conventions
+
+The following document contains a compilation of conventions and guidelines to format, structure and write code for the SDS SDK PY Aspect Model loader.
+
+## General Conventions
+Our code conventions are based on the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html) but
+detailed and adjusted for the needs of the SDS SDK py aspect model loader.
```

Comment:
It is stated here that an adjusted version of the code conventions from Google are used, without explaining what these adjustments are. Please add them or rephrase this statement.